learning_journal/views/default.py

```diff
@@ -0,0 +1,84 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
+from datetime import date as Date
+from sqlalchemy.exc import DBAPIError
+
+from ..models import Entry
+
+
+# @view_config(route_name='home', renderer='../templates/home.jinja2')
+# def my_view(request):
+#     try:
+#         query = request.dbsession.query(Entry)
+#         one = query.filter(Entry.title == 'one').first()
+#     except DBAPIError:
+#         return Response(db_err_msg, content_type='text/plain', status=500)
+#     return {'one': one, 'project': 'learning_journal'}
+
+
+@view_config(route_name='home', renderer='../templates/home.jinja2')
+def home_view(request):
+    """Grab homepage data from db and send it to jinja."""
+    entries = request.dbsession.query(Entry).order_by(Entry.id).all()
+    latest = entries[-1]
+    left_entries, right_entries = [], []
+    for i in range(len(entries) - 1):
+        if i % 2:
+            left_entries.append(entries[i])
+        else:
+            right_entries.append(entries[i])
+    return {'latest': latest,
+            'left_entries': left_entries,
+            'right_entries': right_entries}
```

Comment:
Left / right could be handled by your template.  You can then return just a single list of entries to your template and apply  a left/right class as you iterate through building your index page.